src/user_dto.py

```diff
@@ -28,7 +28,7 @@ class UserDto:
     email: str
     gender: str
     timestamp: str
-    id: str = str(uuid.uuid4())
+    id: str = field(default_factory=lambda: str(uuid.uuid4()))
```

Comment:
What's the reason for this change?

Reply:
This change is responsible for register new user. Older one was generating uuid only once, on server start. Now it's new uuid for every user